cmake/inputs/patches/ep_capnp/capnp_CMakeLists.txt.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/c++/src/capnp/CMakeLists.txt b/c++/src/capnp/CMakeLists.txt
+index deb4becf..a3c0065d 100644
+--- a/c++/src/capnp/CMakeLists.txt
++++ b/c++/src/capnp/CMakeLists.txt
+@@ -186,7 +186,10 @@ if(NOT CAPNP_LITE)
+   install(TARGETS capnp_tool capnpc_cpp capnpc_capnp ${INSTALL_TARGETS_DEFAULT_ARGS})
+ 
+   # Symlink capnpc -> capnp
+-  install(CODE "execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink capnp${CMAKE_EXECUTABLE_SUFFIX} \"\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_BINDIR}/capnpc${CMAKE_EXECUTABLE_SUFFIX}\")")
++#tiledb just exec's capnp directly.  If capnproto project ever changes name
++#of capnp(.exe) may need to reconsider what to do with this, if using the
++#new name is problematic.
++#  install(CODE "execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink capnp${CMAKE_EXECUTABLE_SUFFIX} \"\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_BINDIR}/capnpc${CMAKE_EXECUTABLE_SUFFIX}\")")
```

Comment:
remove dead line

Reply:
This is patching a capnproto module CMakeLists.txt and thus would be removing a line from their capnproto CMakeLists.txt, making it less obvious that something of theirs was being disabled, do you really want me to do that (remove that line)?

Reply:
you're right, let's leave this

Reply:
Yes, the (cmake) symlink attempt was failing on windows.

Reply:
That should work on [CMake 3.13+](https://cmake.org/cmake/help/v3.13/release/3.13.html?highlight=release%20notes#id8). We should probably consider bumping the minimum on Windows to 3.13 or even newer, which may help with some of these compiler flag issues too.

In any case, leaving `resolved`, can revisit later.